packages/app/src/cli/prompts/deploy-release.ts

```diff
@@ -0,0 +1,180 @@
+import metadata from '../metadata.js'
+import {
+  ConfigExtensionIdentifiersBreakdown,
+  ExtensionIdentifiersBreakdown,
+} from '../services/context/breakdown-extensions.js'
+import {InfoTableSection, renderConfirmationPrompt, renderDangerousConfirmationPrompt} from '@shopify/cli-kit/node/ui'
+
+export interface DeployOrReleaseConfirmationPromptOptions {
+  extensionIdentifiersBreakdown: ExtensionIdentifiersBreakdown
+  configExtensionIdentifiersBreakdown?: ConfigExtensionIdentifiersBreakdown
+  appTitle?: string
+  release: boolean
+  force: boolean
+}
+
+export interface DeployConfirmationPromptOptions {
+  appTitle?: string
+  extensionsContentPrompt: {
+    extensionsInfoTable?: InfoTableSection
+    deletedInfoTable?: InfoTableSection
+  }
+  configContentPrompt: {
+    configInfoTable: InfoTableSection
+    deletedInfoTable?: InfoTableSection
+  }
+  release: boolean
+}
+
+export async function deployOrReleaseConfirmationPrompt({
+  force,
+  extensionIdentifiersBreakdown,
+  configExtensionIdentifiersBreakdown,
+  appTitle,
+  release,
+}: DeployOrReleaseConfirmationPromptOptions) {
+  if (force) return true
+  const extensionsContentPrompt = await buildExtensionsContentPrompt(extensionIdentifiersBreakdown)
+  const configContentPrompt = await buildConfigContentPrompt(release, configExtensionIdentifiersBreakdown)
+
+  return deployConfirmationPrompt({
+    appTitle,
+    extensionsContentPrompt,
+    configContentPrompt,
+    release,
+  })
+}
+
+async function deployConfirmationPrompt({
+  appTitle,
+  extensionsContentPrompt: {extensionsInfoTable, deletedInfoTable},
+  configContentPrompt: {configInfoTable, deletedInfoTable: configDeletedInfoTable},
+  release,
+}: DeployConfirmationPromptOptions): Promise<boolean> {
+  const timeBeforeConfirmationMs = new Date().valueOf()
+  let confirmationResponse = true
+
+  let question = `Create a new version${appTitle ? ` of ${appTitle}` : ''}?`
```

Comment:
let question = `${release ? 'Release' : 'Create'} a new version${appTitle ? ` of ${appTitle}` : ''}?`

or easier to read:
const type = release ? 'Release' : 'Create'
const title = appTitle ? ` of ${appTitle}` : ''
let question = `${type} a new version${title}?`

Or if you prefer the current way, i'm ok, just being nitpicky here

Reply:
I had applied this modification in the following [Add modifications to deploy release prompt](https://github.com/Shopify/cli/pull/3205) PR